### DISCUSSION TITLE
how to make bot pathfind to a negative coordinate

### DISCUSSION THREAD

**Discussion by nooberman12121** [PrismarineJS/mineflayer#2352] [Q&A] [answered]
I want to know how to go to a certain coordinate in minecraft (-10000 55 10000) but I have no idea how to get a negative coordinate to work.

How can i do this?

**Comment by amoraschi** [comment, marked as answer]
You use the pathfinder plugin, like:

```js
bot.pathfinder.setGoal(new GoalBlock(-10000, 55, 10000))
```

Same with positive numbers, please check out the examples
